src/style.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  background: rgb(122, 69, 53);
+  background: linear-gradient(298deg, rgba(122, 69, 53, 1) 17%, rgba(110, 108, 48, 0.9639756944444444) 61%);
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
+h1 {
+  text-align: center;
+  font-size: 40px;
+}
+
+.container {
+  display: flex;
+  margin: 10% 30% 0 30%;
+  flex-direction: column;
+  background-color: rgb(134, 40, 40);
+  border-radius: 5px;
+}
+
+.container h2 {
+  text-align: center;
+  padding-left: 12px;
+  margin-bottom: 10px;
+}
+
+.container h2::after {
+  content: '';
+  display: block;
+  position: relative;
+  width: 100%;
+  border-bottom: 2px solid rgba(252, 252, 252, 0.6);
+  margin-top: 0.5rem;
+  margin-left: -10px;
+}
+
+.add-input {
+  border: none;
+  margin: 0;
+  height: 2rem;
+  padding: 2%;
+}
+
+ul {
+  padding: 0;
+  margin: 0 0 0 2%;
+  list-style: none;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 2px solid rgba(180, 0, 0, 0.6);
+}
+
+.list {
+  display: flex;
+}
+
+.list input {
+  cursor: pointer;
+}
+
+.trash {
+  margin-right: 2%;
+  cursor: pointer;
+}
+
+.fa-ellipsis {
+  margin-top: 1.25rem;
+  rotate: 90deg;
+}
+
+.clearBtn {
+  border: none;
+  height: 4rem;
+  font-size: 1rem;
+  color: black;
+  transition: 0.2s;
+  background-color: chartreuse;
+  cursor: pointer;
+}
+
+.clearBtn:hover {
+  background-color: cadetblue;
+}
```

Comment:
- ***[OPTIONAL]*** 
![image](https://github.com/MariamOsman11/todo-list/assets/60083437/39c4f338-952d-4cc0-85b6-c781e53ad60d)
- [ ] Here I can see you have used three colors that have too much contrast and are not suitable for the eyes. It is very hard for me to read the text. One is `gradient for background` another is `red for tasklist` and the last one is `green for clear button`. I would recommend you to use a `light` color, such as the design given in the project requirement.